toolkit/tools/imagegen/configuration/systemconfig.go

```diff
@@ -26,6 +26,7 @@ type SystemConfig struct {
 	Groups             []Group             `json:"Groups"`
 	Users              []User              `json:"Users"`
 	Encryption         RootEncryption      `json:"Encryption"`
+	AutoRemoveRpmDb    bool                `json:"AutoRemoveRpmDb`
```

Comment:
Please also update the test configuration in `configuration_test.go` with this field so its serialization / deserialization is tested.